examples/applications/author-disambiguation/sampling.py

```diff
@@ -0,0 +1,255 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Beard.
+# Copyright (C) 2015 CERN.
+#
+# Beard is a free software; you can redistribute it and/or modify it
+# under the terms of the Revised BSD License; see LICENSE file for
+# more details.
+
+"""Script for generating the training set.
+
+It samples pairs of signatures labled with 1 if they are of different authors
+or 0 if they are of the same author.
+
+.. codeauthor:: Hussein Al-Natsheh <hussein.al.natsheh@cern.ch>
+.. codeauthor:: Mateusz Susik <mateusz.susik@cern.ch>
+"""
+
+from __future__ import print_function
+
+import argparse
+import json
+import math
+import numpy as np
+import random
+
+from sklearn.cross_validation import train_test_split
+
+from beard.clustering import block_double_metaphone
+from beard.clustering import block_last_name_first_initial
+
+
+def pair_sampling(blocking_function,
+                  blocking_threshold,
+                  clusters_filename,
+                  balanced, verbose,
+                  train_filename,
+                  sample_size,
+                  use_blocking):
```

Comment:
Can we set good default values to balanced, verbose, sample_size and use_blocking?